data/a-a.ts

```diff
@@ -0,0 +1,32 @@
+// TODO
+const TEST_CFR = {};
+
+export default {
+  name: "A/A test",
+  description: "Use this to test audiences or to diagnose issues in the system.",
+  presets: {
+    // Specific constraints for the ui
+    max_bucket_count: 100,
+
+    // Defaults for specific experiment values
+    branches:  [
+      {
+        slug: "control",
+        is_reference_branch: true,
+        ratio: 1,
+        value: TEST_CFR,
+      },
+      {
+        slug: "treatment",
+        ratio: 1,
+        value: TEST_CFR,
+      }
+    ],
+
+    bucketing: {
+      randomization_id: "normandy_id",
```

Comment:
Not sure these are all the same, but should this be `randomization_id` or `randomization_unit`, per https://github.com/mozilla/rapid-experiments-shared/pull/19/files#diff-abab96a820f827ebad3b6a423a30b3f3R35-R44:

```typescript
interface BucketConfig {
  randomization_unit: "client_id" | "normandy_id";
  namespace: string;
  start: number;
  count: number;
  /**
   * Total number of buckets
   * @default  */
  total: number;
}
```